src/main/java/us/dot/its/jpo/ode/aws/depositor/AwsDepositor.java

```diff
@@ -482,7 +394,7 @@ private JSONObject generateAWSProfile() throws IOException {
 	}
 
 	private static String getEnvironmentVariable(String variableName) {
-		String value = System.getenv(variableName);
+		String value = System.getProperty(variableName);
```

Comment:
Why the switch to getProperty here?

Reply:
This has been changed back to System.getenv to access environment variables instead of system properties.